cartridges/int_affirm_controllers/cartridge/templates/default/affirm/paymentmethodinput.isml

```diff
@@ -29,10 +29,9 @@
 			<img alt="${Resource.msg(paymentMethodType.label,'forms',null)}" 
 				src="https://cdn-assets.affirm.com/images/blue_logo-transparent_bg.png" 
 				title="${Resource.msg(paymentMethodType.label,'forms',null)}"/>
-			<span>${Resource.msg('payment.monthlypayments', 'affirm', null)}</span>
-			<a class="affirm-product-modal" style=""
-			data-amount="${pdict.Basket.totalGrossPrice.multiply(100).getValue().toFixed()}"
-			>${Resource.msg('payment.learnmore', 'affirm', null)}</a>
```

Comment:
was there a reason why we got rid of that?

Reply:
Restored the class